feature/dashboard/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/dashboard/ui/SurveyHint.kt

```diff
@@ -0,0 +1,186 @@
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+private const val SURVEY_URL = "https://example.com/survey"     // TODO
+
+@Composable
+fun SurveyHint(
+    modifier: Modifier = Modifier,
+    surveyHintService: SurveyHintService
+) {
+    val shouldDisplaySurveyHint by surveyHintService.shouldShowSurveyHint.collectAsState(initial = false)
+    var displaySurveyHint by rememberSaveable { mutableStateOf(false) }
+    var showSurveyDialog by rememberSaveable { mutableStateOf(false) }
+
+    LaunchedEffect(shouldDisplaySurveyHint) {
+        if (shouldDisplaySurveyHint) displaySurveyHint = true
+    }
+
+    AnimatedVisibility(displaySurveyHint) {
+        SurveyHintImpl(
+            modifier = modifier,
+            onClick = { showSurveyDialog = true }
+        )
+    }
+
+    val uriHandler = LocalUriHandler.current
+    val scope = rememberCoroutineScope()
+    if (showSurveyDialog) {
+        SurveyDialog(
+            onClose = { participate ->
+                if (participate) {
+                    scope.launch {
+                        uriHandler.openUri(SURVEY_URL)
+                        surveyHintService.dismissSurveyHintPermanently()
+
+                        delay(2000) // Wait for the survey to open before hiding the hint
+                        displaySurveyHint = false
+                    }
+                }
+
+                showSurveyDialog = false
+            }
+        )
+    }
+}
+
+@Composable
+private fun SurveyHintImpl(
+    modifier: Modifier,
+    onClick: () -> Unit
+) {
+    Card(
+        modifier = modifier,
+        onClick = onClick
+    ) {
+        Row(
+            modifier = Modifier.padding(16.dp),
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Icon(
+                imageVector = Icons.Default.Info,
+                contentDescription = null,
+                tint = MaterialTheme.colorScheme.primary
+            )
+
+            Spacer(modifier = Modifier.width(8.dp))
+
+            Text(
+                text = stringResource(R.string.survey_hint_text),
+            )
+
+            Spacer(modifier = Modifier.weight(1f))
+
+            Icon(
+                imageVector = Icons.Default.ChevronRight,
+                contentDescription = null,
+                tint = MaterialTheme.colorScheme.primary
+            )
+        }
+    }
+}
+
+
+@Composable
+private fun SurveyDialog(
+    modifier: Modifier = Modifier,
+    onClose: (participate: Boolean) -> Unit
+) {
+
+    AlertDialog(
+        onDismissRequest = { onClose(false) },
+        title = { Text(text = stringResource(R.string.survey_dialog_title)) },
+        text = {
+            Column(
+                modifier = Modifier,
+                verticalArrangement = Arrangement.spacedBy(32.dp)
+            ) {
+                Text(text = stringResource(R.string.survey_dialog_do_you_have_time))
```

Comment:
I think you could add `textAlign = TextAlign.Center` to the center the texts in the dialog.

Reply:
I think it's fine to stick to the android standards here :)